data-files/scene/Simple_Hallway.Scene.Any

```diff
@@ -17,8 +17,11 @@
         
 		player = PlayerEntity {
             model = "playerModel";
-            frame = Point3(47.522999, -2.3549, -0.35916	);
-			heading = -1.5707963267948966192313216916398;
+            // When the player entity is specified, its frame overrides the camera frame
+            // even if not specified and filled with the default.
+            // If you prefer to use a specified camera frame, you can specify which camera to use
+            // in the experiment config.
+            frame = CFrame::fromXYZYPRDegrees(46.08, -2.2, 0.0, -90, 0, 0 ); 
```

Comment:
It seems to me that `46.0` in the x is more likely to be centered than `46.08` like it is now.

Reply:
If you think this is correct I'm fine w/ changing it. Since we are changing the spawn position from past behavior here, we might as well pick the most "correct" option possible. I do not have strong opinions on this spawn position, and we can always refer to older scene configs to recover the historical spawn position.